src/autoAuthenticate.ts

```diff
@@ -24,7 +28,7 @@ export const useAutoAuthenticate = (): AutoAuthenticateState => {
         ) {
             console.debug("auto-authenticating");
             setLSNotSignedIn();
-            setIsAutoAuthenticating(true);
+            dispatch(setIsAutoAuthenticating(true));
             performAuth().catch(console.error);
```

Comment:
should it be set back to `false` in a `finally` call?

Reply:
true, although I think performAuth redirects? so if the promise is successful I don't think it should, since I want to keep the state `true` until it refreshes. I'll add a reset for the error condition though.